src/view/com/util/post-embeds/QuoteEmbed.tsx

```diff
@@ -127,14 +130,15 @@ export function QuoteEmbed({
         <PostAlerts moderation={moderation} style={styles.alert} />
       ) : null}
       {richText ? (
-        <RichText
-          enableTags
-          value={richText}
-          style={[a.text_md]}
-          numberOfLines={20}
-          disableLinks
-          authorHandle={quote.author.handle}
-        />
+        <View pointerEvents="none">
+          <RichText
+            enableTags
+            value={richText}
+            style={[a.text_md]}
+            numberOfLines={20}
+            authorHandle={quote.author.handle}
+          />
+        </View>
```

Comment:
Looks like this doesn't work on web, may be because Radix is using event delegation 🤔

Reply:
Changed the fix to work on both platforms 👍 Think you were onto something.